pyproject.toml

```diff
@@ -1,5 +1,172 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools>=61.0", "setuptools-scm[git]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "eodatasets3"
+description = "Packaging, metadata and provenance for OpenDataCube EO3 datasets"
+readme = "README.md"
+requires-python = ">=3.10"
+license = "Apache-2.0"
+authors = [{name = "Open Data Cube"}]
+dynamic = ["version"]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Operating System :: OS Independent",
+]
+
+# Core dependencies (formerly install_requires)
```

Comment:
The item on the next line is called `dependencies`, so not sure this comment adds much value.

Reply:
Agreed, gone.